### PR TITLE
Security Hub Region customization

### PR DESCRIPTION
**Pull request by danielmkn**
Change the value of Region and ProductArn field to match the region of Security hub.

Add try-catch block for importing findings and recording to DB.

### CODE REVIEW

File: transformer/app.js [hunk 1]
```diff
@@ -225,11 +225,18 @@ export async function lambdaHandler(event, context) {
...
-      hubImportResponse = await hubClient.send(new BatchImportFindingsCommand({ Findings: newFindingsToImport }));
-      logger.debug('Security Hub import response', { hubImportResponse });
-
-      const dbResponse = await writeFindingsToDB(newFindingsToImport);
-      logger.debug('DB response', { dbResponse });
+      try {
+        hubImportResponse = await hubClient.send(new BatchImportFindingsCommand({ Findings: newFindingsToImport }));
+        logger.debug('Security Hub import response', { hubImportResponse });
+      } catch (e) {
+        logger.debug(`Error while importing findings ${e}`);
```
Comment (alexhung) [inline transformer/app.js:232]: Please use `logger.error()`

File: transformer/app.js [hunk 2]
```diff
@@ -225,11 +225,18 @@ export async function lambdaHandler(event, context) {
...
+      }
+      try {
+        const dbResponse = await writeFindingsToDB(newFindingsToImport);
+        logger.debug('DB response', { dbResponse });
+      } catch (e) {
+        logger.debug(`Error while importing findings ${e}`);
```
Comment (alexhung) [inline transformer/app.js:238]: Same here with `logger.error()`.

File: transformer/app.js (hunk 1 above)
Comment (alexhung) [reply transformer/app.js:232]: Let's not use string interpolation here. Use the same pattern as other `logger` calls, e.g. `logger.error('Error while importing findings', { e })`.

**Review by alexhung — CHANGES_REQUESTED** [review]
I thought we need to set the `SECURITY_HUB_REGION` to the `SecurityHubClient`? Not seeing that change.

**Comment by danielmkn** [comment]
The region config was already there on line 11.

File: transformer/app.js (hunk 1 above)
Comment (danielmkn) [reply transformer/app.js:232]: fixed

File: transformer/app.js (hunk 2 above)
Comment (danielmkn) [reply transformer/app.js:238]: fixed

**Review by alexhung — APPROVED** [review]

**Pull request merged**